docs/networking/rocm.rst

```diff
@@ -0,0 +1,119 @@
+ROCm
+====
+
```

Comment:
You might want to have a sentence or three defining what ROCm is.  Feel free to link out to web sites that provide definitions (e.g., AMD-maintained sites).  You might want to pick web sites that will assumedly be long-lived (and not disappear if a product SKU changes or somesuch).